andela/casablanca-cfh
# #156096161 Redesign UI for authentication screen

#### What does this PR do?
 It redesigns the authentication pages(signup/signin)

#### Description of Task to be completed?
Re-Design UI for user authentication screen

#### How should this be manually tested?

- clone the repo and change into the project directory
- run `npm install` to install required packages
- create a `.env` file and add all the required environment variables as specified in `.env.example`
- use `npm run start:dev` to start up the server
- visit `http://localhost:3000/#!/signup` to see and test the signup page
- visit `http://localhost:3000/#!/signin` to see and test the signin page

#### Any background context you want to provide?
Note that since the main gaming page is not ready, after signining in or signing up, it would redirect back to the home page.
#### What are the relevant pivotal tracker stories?
#156096161
#### Screenshots
![screencapture-localhost-3000-2018-04-08-17_39_51](https://user-images.githubusercontent.com/26048536/38469906-e5f06078-3b53-11e8-8c7a-279fafafaedf.png)
![screencapture-localhost-3000-2018-04-07-08_06_02](https://user-images.githubusercontent.com/26048536/38452351-97c04c48-3a3a-11e8-9a67-af19a33412fb.png)
